bin/s2nc.c

```diff
@@ -45,6 +45,11 @@
 #define OPT_PREFER_THROUGHPUT  1006
 #define OPT_BUFFERED_SEND      1007
 
+/*
+ * s2nc is an example client that uses many s2n-tls APIs. It's main purpose is to provide
+ * testing functionality for s2n-tls integration tests, and as such, it should not be used
+ * for any production purposes.
+ */
```

Comment:
It's not only used for the integ tests-- it's also super useful to try to repro failures or ad hoc testing.
```suggestion
/*
 * s2nc is an example client that uses many s2n-tls APIs.
 * It is intended for testing purposes only, and should not be used in production.
 */
```